public/app/plugins/datasource/cloudwatch/language_provider.test.ts

```diff
@@ -0,0 +1,171 @@
+import { Value } from 'slate';
+import { TypeaheadOutput } from '@grafana/ui';
+import { CloudWatchDatasource } from './datasource';
+import { GetLogGroupFieldsResponse } from './types';
+import { CloudWatchLanguageProvider } from './language_provider';
+import Prism, { Token } from 'prismjs';
+import {
+  AGGREGATION_FUNCTIONS_STATS,
+  BOOLEAN_FUNCTIONS,
+  DATETIME_FUNCTIONS,
+  FUNCTIONS,
+  IP_FUNCTIONS,
+  NUMERIC_OPERATORS,
+  QUERY_COMMANDS,
+  STRING_FUNCTIONS,
+} from './syntax';
+
+const fields = ['field1', '@message'];
+
+describe('CloudWatchLanguageProvider', () => {
+  it('should suggest ', async () => {
+    await runSuggestionTest('stats count(\\)', [fields]);
```

Comment:
The cursor is marked with `\\` because `|` is a regular query element and I think `/` is valid in the field names.